### ISSUE TITLE
instructor user -- no way to set?  

### ISSUE THREAD

**Issue by hanleybrand** [instructure/ForgeLTI#5]
I have ForgeLTI running on a local server (in tandem with a Blackboard Learn developer vm) and while I can get basic registration to work, the instructor user gets set (I think) as a student, even though I have the LTI launch set to send user roles.

Is there any way to fix the registration in the db once it's set? I saw that the User class has a boolean attribute "Instructor" but there doesn't seem to be a command to flip that bit. 

I'm setting up a dev environment now to have a closer look, but I thought I'd mention it here, since I'm not sure if this issue is Learn specific or if it's more general.

**Comment by bluej100** [comment]
It just uses a SQLite db, so you could use that if you know SQL, but I'd be
happy to add an op command to make a user a teacher as well. I'm glad
you're trying it out and that it works with Blackboard!

**Comment by bluej100** [comment]
Man, I forgot that I switched it from SQLite to H2. Never mind. I'll add that command.

**Comment by hanleybrand** [comment]
Thanks - I had discovered that but forgot to come back and mention it ;-)  

I have intellij, but couldn't get it's database tool to interact with the H2 tables that the plugin creates, not sure why (it can query the MODS.INFORMATION_SCHEMA database and see a MODS.PUBLIC database -- but I'm deducing that there should be at least a table for Assignment, LTIConsumer and User, right?)

**Comment by bluej100** [comment]
Yeah, that's strange. Well, I'll get that command added and release a new version tonight. Thanks for your patience.

**Comment by bluej100** [comment]
I've implemented the feature in BukkitLTI and released a new version of the plugin. Before I port it to ForgeLTI, do you have a preference for whether I leave this mod on 1.7.10 or upgrade to 1.8.9?

**Comment by bluej100** [comment]
For what it's worth, I was successfully set as an instructor when registering from Canvas, and I believe that was also working in Sakai at one point. I'd be interested to hear what Blackboard is sending; my check is pretty simple: https://github.com/instructure/MinecraftLTI/blob/master/src/main/java/com/instructure/minecraftlti/LTIServlet.java#L47 .
